runtime/gc_base/GCExtensions.hpp

```diff
@@ -308,6 +308,28 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	 * @return true if we need to scan the java stack
 	 */
 	static bool needScanStacksForContinuationObject(J9VMThread *vmThread, j9object_t objectPtr, bool isConcurrentGC, bool isGlobalGC, bool beingMounted);
+#if JAVA_SPEC_VERSION >= 19
+	/*
+	 *
+	 * param[in] checkConcurrentState can be J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_LOCAL or J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_GLOBAL
+	 *
+	 *  There is no need scanning before continuation is started or after continuation is finished.
+	 *  If WinningConcurrentGCScan set J9_GC_CONTINUATION_bit3:STATE_CONCURRENT_SCAN_LOCAL or bit4:J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_GLOBAL in the state base on checkConcurrentState
+	 *	If low tagging(bit3 or bit4) failed due to either
+	 *
+	 *   a carrier thread winning to mount, we don't need to do anything, since it will be compensated by pre/post mount actions
+	 *   another GC thread winning to scan(bit3/bit4,bit3 and bit4 is irrelevant and independent), again don't do anything, and let the winning thread do the work, instead
+	 */
+	static bool tryWinningConcurrentGCScan(ContinuationState volatile *continuationStatePtr, bool isGlobalGC, bool beingMounted);
+
+	/**
+	 * clear CONCURRENTSCANNING flag bit3:for LocalConcurrentScanning /bit4:for GlobalConcurrentScanning base on isGlobalGC,
```

Comment:
no need to go into details of specific bits in comments (check other comments too)